### ISSUE TITLE
pickadate preventing click events on disabled dates after selecting other date

### ISSUE THREAD

**Issue by renatodeleao** [amsul/pickadate.js#681]
This is a quite confusing title, but it's 3am and I still can't figure how to do this.

*tl;dr* -- > http://codepen.io/ohsimtabem/pen/zGOxKO

Let's say I'm using pickadate as calendar to display some events. So i'm preventing it from close, and making it always visible. On start I `set('disable', [disableDates])` the dates of my events.

When a user clicks on one of this events(`picker__day--disabled`) some action is triggered, for the sake of explanation, let's say an `alert`. Everything goes well until i select another random date and click back again on one of those disabled dates: the alert isn't triggered anymore. Is there anyway to workaround this? What am i missing?

Dispite that, still the best open source datepicker available. Thanks in advance.

**Comment by mdisconzi** [comment]
Hi,

For some reason, the pickadate render event, "clean" the handler onclick added, just add to   picker__day--disable on render event, like this:

`$('pickadate obj').pickadate('picker').on({render:function(){
         $('.picker__day--disabled').on('click',function(ev){
            //do something
         });
});`

**Comment by amsul** [comment]
@renatodeleao it seems as though the pen has been deleted. Still having trouble with this or is it safe to close?

**Comment by renatodeleao** [comment]
@amsul sry for the delay, not sure what happened to the pen.. but well it was a year go so I might have done a workaround for it ^ probably using @mdisconzi advice (thanks).

Are you back on dev with this or just passing by? 


(closing)
